lauren_vicknair/homework/homework3/homework3.rb

```diff
@@ -0,0 +1,114 @@
+#1
+
+rand_array = [12313, 232, 22342, 2342421, 131231, 354353, 677685, 3422332, 97867876]
+
+def my_max(nums)
+   current_max = 0
+   nums.each do |x|
+       if x > current_max
+           current_max = x
+       end
+   end
+   return current_max
+end
+
+puts my_max(rand_array)
+
+
+
+
+require 'httparty'
+require 'json'
+
+def search_itunes(term)
+ base = "https://itunes.apple.com/search?term="
```

Comment:
@LCVicknair It's a good idea to get into the habit of consistently indenting code. Ruby people prefer two spaces per indentation. It's great when you can spot errors like missing `end` statements for example with a simple glance.